packages/components/src/number-control/stories/index.tsx

```diff
@@ -62,3 +62,35 @@ export const Default = Template.bind( {} );
 Default.args = {
 	label: 'Value',
 };
+
+// Check if this was broken and at what point
+// in particular on commit actions, when `min` is not `undefined`
+export const Uncontrolled: ComponentStory< typeof NumberControl > = ( {
```

Comment:
Please ignore the code comment, it's a left over from a copy-paste.

The whole Storybook example will be removed before merging, I'm just leaving around for ease of testing